src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java

```diff
@@ -73,7 +88,12 @@ void shouldReturnTemplateContentForSole() {
             entry(DATE_OF_HEARING, "8 November 2021"),
             entry(TIME_OF_HEARING, "14:56 pm"),
             entry(BEFORE_DATE_OF_HEARING, "1 November 2021"),
-            entry(HAS_FINANCIAL_ORDERS, true)
+            entry(HAS_FINANCIAL_ORDERS, true),
+            entry(CTSC_CONTACT_DETAILS, buildCtscContactDetails()),
+            entry(DIVORCE_AND_DISSOLUTION_HEADER, "Divorce and Dissolution"),
+            entry(COURTS_AND_TRIBUNALS_SERVICE_HEADER, "HM Courts & Tribunals Service"),
+            entry(CONTACT_EMAIL, "contactdivorce@justice.gov.uk"),
+            entry(PHONE_AND_OPENING_TIMES, "Phone: 0300 303 0642 (Monday to Friday, 8am to 6pm)")
```

Comment:
use PHONE_AND_OPENING_TIMES_TEXT

Reply:
Removed magic strings and replaced with constants